integration/exec_test.go

```diff
@@ -12,6 +12,17 @@ import (
 )
 
 func TestExecPS(t *testing.T) {
+	testExecPS(t, false)
+}
+
+func TestUsernsExecPS(t *testing.T) {
+	if _, err := os.Stat("/proc/self/ns/user"); os.IsNotExist(err) {
```

Comment:
Yeah, I know, but other namespaces can be disabled in kernel too. Why `NEWUSER` is special? I think it is probably better to say that if you want to run tests - you should use proper kernel config, because I think `t.Skip`s can obscure some errors from developer and we will have to deal with them in our CI.